The `useAsyncCallback` test often fails

### Which package has an issue?

@yamada-ui/react

### Package Version

1.6.0

### Description

The `useAsyncCallback` test often fails.

```
 ❯ packages/hooks/use-async-callback/tests/index.test.tsx:193:11
    191|     expect(
    192|       document.querySelector(".ui-loading-background"),
    193|     ).not.toBeInTheDocument()
       |           ^
    194|     expect(el).not.toBeDisabled()
 Test Files  1 failed | 161 passed (162)
    195|     expect(mockCallback).toHaveBeenCalledWith()

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯
      Tests  1 failed | 1622 passed | 2 skipped | 1 todo (1626)
   Start at  00:49:01
   Duration  291.39s (transform 6.47s, setup 52.70s, collect 321.22s, tests 287.99s, environment 88.68s, prepare 18.06s)



Error: Error: expect(element).not.toBeInTheDocument()

expected document not to contain element, found <div
  class="ui-loading-background css-1dzfmgq"
  style="opacity: 0.005394638565577203; will-change: opacity, transform; transform: scale(0.9502697319282788);"
>
  <svg
    class="ui-icon ui-loading css-uzzcv7"
    viewBox="0 0 38 38"
  >
    <g
      fill="none"
      fill-rule="evenodd"
    >
      <g
        stroke-width="2"
        transform="translate(1 1)"
      >
        <circle
          cx="18"
          cy="18"
          r="18"
          stroke-opacity=".5"
        />
        <path
          d="M36 18c0-9.94-8.06-18-18-18"
        >
          <animatetransform
            attributeName="transform"
            dur="1s"
            from="0 18 18"
            repeatCount="indefinite"
            to="360 18 18"
            type="rotate"
          />
        </path>
      </g>
    </g>
  </svg>
</div> instead
 ❯ packages/hooks/use-async-callback/tests/index.test.tsx:193:11


 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```


### Link to Reproduction

https://github.com/yamada-ui/yamada-ui/actions/runs/11622216187/job/32367327545

### Steps to Reproduce

1. Go to '...'
2. Click on '...'
3. Scroll down to '...'
4. See error


### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

--- a/packages/hooks/use-async-callback/tests/index.test.tsx
+++ b/packages/hooks/use-async-callback/tests/index.test.tsx
@@ -1,6 +1,6 @@
 import type { ThemeConfig } from "@yamada-ui/core"
 import type { FC } from "react"
-import { render, renderHook, screen } from "@yamada-ui/test"
+import { render, renderHook, screen, waitFor } from "@yamada-ui/test"
 import { vi } from "vitest"
 import { useAsyncCallback } from "../src"
 
@@ -111,7 +111,9 @@ describe("useAsyncCallback", () => {
     await wait(500)
     rerender(<Component />)
     expect(result.current[0]).toBeFalsy()
-    expect(document.querySelector(".ui-loading-page")).not.toBeInTheDocument()
+    await waitFor(() => {
+      expect(document.querySelector(".ui-loading-page")).not.toBeInTheDocument()
+    })
     expect(el).not.toBeDisabled()
     expect(mockCallback).toHaveBeenCalledWith()
   })
@@ -147,7 +149,9 @@ describe("useAsyncCallback", () => {
     await wait(500)
     rerender(<Component />)
     expect(result.current[0]).toBeFalsy()
-    expect(document.querySelector(".ui-loading-page")).not.toBeInTheDocument()
+    await waitFor(() => {
+      expect(document.querySelector(".ui-loading-page")).not.toBeInTheDocument()
+    })
     expect(el).not.toBeDisabled()
     expect(mockCallback).toHaveBeenCalledWith()
   })
@@ -188,9 +192,11 @@ describe("useAsyncCallback", () => {
     await wait(500)
     rerender(<Component />)
     expect(result.current[0]).toBeFalsy()
-    expect(
-      document.querySelector(".ui-loading-background"),
-    ).not.toBeInTheDocument()
+    await waitFor(() => {
+      expect(
+        document.querySelector(".ui-loading-background"),
+      ).not.toBeInTheDocument()
+    })
     expect(el).not.toBeDisabled()
     expect(mockCallback).toHaveBeenCalledWith()
   })
@@ -229,7 +235,9 @@ describe("useAsyncCallback", () => {
     await wait(500)
     rerender(<Component />)
     expect(result.current[0]).toBeFalsy()
-    expect(document.querySelector(".ui-loading-page")).not.toBeInTheDocument()
+    await waitFor(() => {
+      expect(document.querySelector(".ui-loading-page")).not.toBeInTheDocument()
+    })
     expect(el).not.toBeDisabled()
     expect(mockCallback).toHaveBeenCalledWith()
   })

